ui/client/components/graph/node-modal/NodeDetailsContent.js

```diff
@@ -537,7 +537,7 @@ export class NodeDetailsContent extends React.Component {
       (newValue) => this.setNodeDataAt(fieldProperty, newValue, defaultValue),
       readonly,
       this.isMarked(fieldProperty),
-      key,
+      key || fieldProperty || fieldLabel,
```

Comment:
Why is it needed here?

Reply:
wanted some place with "old" value. there is "*" postfix in renamed (not yet saved) value. I think it's nice feature to show this as hint and only old value is enough in my opinion

Reply:
every iteration in react needs key's for elements. in this function key is the last argument and it's most time omitted in usages. I just added some fallback to almost always set and probably unique values to avoid react warnings.